### PR TITLE
Added all eda scripts and images

### PR DESCRIPTION
**Pull request by Arushi282**

### CODE REVIEW

**Comment by Arushi282** [comment]
I forgot to mention this but I also added new eda diagrams in the eda jupyter notebook, I still have to write a small paragraph for each diagram

**Comment by liannah** [comment]
I think it would have been better structure wise to have functions for plot creations and then call them in main. But it is just my opinion, @Arushi282 will that complicate the task for you ? If yes, that is okay to have It this way, but I guess some comments throughout will be needed, as well as assert statements to check that the input is of correct type something like     `assert isinstance(train_df_viz, pd.DataFrame, "The input is not a dataframe")` or for instance that the second column name is equal to the one we are expecting so to make sure that we are getting the right data frame.  
.

**Comment by Arushi282** [comment]
> I think it would have been better structure wise to have functions for plot creations and then call them in main. But it is just my opinion, @Arushi282 will that complicate the task for you ? If yes, that is okay to have It this way, but I guess some comments throughout will be needed, as well as assert statements to check that the input is of correct type something like `assert isinstance(train_df_viz, pd.DataFrame, "The input is not a dataframe")` or for instance that the second column name is equal to the one we are expecting so to make sure that we are getting the right data frame. .

I will add comments on what my code is doing, thanks lianna!

File: src/random_search_eda.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# author: Arushi Ahuja
+# date: 2021-11-24
+
+'''This script generates all the EDA diagrams used in the final report 
+  after model generation, specifically the random search diagram
+
```
Comment (thayeylolu) [inline src/random_search_eda.py:6]: The description is the same as `credit default eda.py` . 😄

**Review by thayeylolu — CHANGES_REQUESTED** [review]

File: src/random_search_eda.py (hunk 1 above)
Comment (Arushi282) [reply src/random_search_eda.py:6]: No I changed it to "after model generation"

**Comment by Arushi282** [comment]
Okay so I made all the changes necessary, Additionally I added all my written analysis in the Rmd report only, and also wrote some analysis in the eda jupyter notebook

**Pull request merged**